tee-worker/cli/demo_shielding_unshielding.sh

```diff
@@ -64,17 +64,24 @@ echo "Using node uri ${NODEURL}:${NPORT}"
 echo "Using trusted-worker uri ${WORKER1URL}:${WORKER1PORT}"
 echo ""
 
-# the parachain LIT is 12 decimal
+# the parentchain token is 12 decimal
 UNIT=$(( 10 ** 12 ))
 
-# we have to make these amounts greater than ED, see
-# https://github.com/litentry/litentry-parachain/issues/1162
```

Comment:
Let's keep the issue link

Reply:
Similar to this comment: https://github.com/integritee-network/worker/blob/f124aa620036fa0d7c7f1b05175dfb6c89f3c968/core/parentchain/indirect-calls-executor/src/lib.rs#L177